driver/src/main/scala/org/mongodb/scala/WriteConcern.scala

```diff
@@ -104,89 +107,11 @@ object WriteConcern {
   def apply(w: Int, wtimeout: Int): JWriteConcern = new JWriteConcern(w, wtimeout)
```

Comment:
Should be:

    def apply(w: Int, wTimeoutMS: Int):

since the unit is not specified otherwise.  And the 't' should be capitalized to be consistent with the Java driver and the other apply method.